LICENSE.txt

```diff
@@ -1646,7 +1646,7 @@ EqualsVerifier | release normal jar (3.7.2)
 * Declared Licenses *
 Apache-2.0
 
-Copyright 2022, EqualsVerifier | release normal jar Contributors
```

Comment:
Please leave the copyrights of our dependencies w/o change. Only the first change in this file (GD copyright) should be updated